vaccine/application.py

```diff
@@ -43,30 +118,213 @@ async def check_age(content: Optional[str]):
                 age = int(content or "")
                 assert age >= 0
             except (ValueError, TypeError, AssertionError):
-                raise ErrorMessage(
-                    "Sorry, we don't understand your response. "
-                    "Please type the number that represents your age in years"
-                )
+                raise ErrorMessage("⚠️  Reply using numbers only. Example *27*")
 
         return FreeText(
             self,
-            question="What is your current age, in years?",
-            next="state_end",
+            question="\n".join(
+                ["◼️◼️◼️◻️◻️", "", "How old are you? (Reply with a number)"]
+            ),
+            next="state_location",
             check=check_age,
         )
 
-    async def state_end(self):
+    async def state_location(self):
+        return FreeText(
+            self,
+            question="\n".join(
+                [
+                    "◼️◼️◼️◼️◻️",
+                    "",
+                    "Where do you live? (Reply with name of suburb, town, settlement "
+                    "or nearest)",
+                ]
+            ),
+            next="state_comorbidities",
+        )
+
+    async def state_comorbidities(self):
+        not_sure = "\n".join(
+            [
+                "Having one or more specific chronic conditions could impact which "
+                "phase you are in. These conditions include:",
```

Comment:
Do you know whether this is the full list? Just thinking to confirm with Neville

Reply:
This content is from Neville, but will confirm

Reply:
These are the current co-morbidities approved by NDOH